vscode_extension/src/commands/toggleUntypedCodeHighlighting.ts

```diff
@@ -34,12 +47,12 @@ function backwardsCompatibleValue(
 /**
  * Toggle highlighting of untyped code.
  * @param context Sorbet extension context.
- * @returns `true` if highlighting is now enabled, `false` otherwise.
+ * @returns The new TrackUntyped setting
```

Comment:
Not needed, but it makes it a link in docs.

```suggestion
 * @returns The new {@link TrackUntyped} setting.
```